shared/scss/popup.scss

```diff
@@ -666,6 +666,8 @@ body {
     }
 }
 
+@import "views/privacy-practices";
```

Comment:
I added this new `views` folder where we can place styles that are specific to views. It feels messy to have them all in the bigger files. What do you think?

Also, I tried to minimize nesting as much as possible, to get the full benefit out of using BEM. Should probably try and enforce that elsewhere.